charts/consul/values.yaml

```
@@ -485,17 +489,24 @@ server:
   #     -additional-dnsname="server.{{datacenter}}.{{domain}}"
   # ```
   #
-  # If you have generated the
-  # server-cert yourself with the consul CLI, you could use the following command
+  # If you have generated the server-cert yourself with the consul CLI, you could use the following command
   # to create the secret in Kubernetes:
   #
   # ```bash
   # kubectl create secret generic consul-server-cert \
   #     --from-file='tls.crt=./dc1-server-consul-0.pem'
   #     --from-file='tls.key=./dc1-server-consul-0-key.pem'
   # ```
+  #
+  # Vault Secrets backend:
+  # If you are using Vault as a secrets backend a Vault Policy must be created which allows `["create", "update"]`
+  # capabilities on the PKI issuing endpoint, which is usually of the form `pki_int/issue/consul-server`. 
```

to make the mount path consistent with the comment about ca
```suggestion
  # capabilities on the PKI issuing endpoint, which is usually of the form `pki/issue/consul-server`. 
```